src/Template/Sentences/search.ctp

```diff
@@ -86,6 +86,19 @@ if ($ignored) {
 <section layout="row" ng-cloak>
 
 <md-content class="md-whiteframe-1dp" flex>
+
+<md-toolbar class="md-hue-2" hide-gt-xs ng-cloak ng-controller="SidenavController">
+    <div class="md-toolbar-tools">
+        <h2 flex><?= $this->Pages->formatResultCount($this->Paginator, $real_total); ?></h2>
+
+        <md-button ng-click="toggle('advanced-search')">
+            <md-icon>filter_list</md-icon>
+            <?php /* @translators: button to open the advanced search option sidebar on mobile */ ?>
+            <?= __('Narrow search') ?>
```

Comment:
That looks pretty cool! :sunglasses: But now I can’t see how to open the search criteria sidebar when the width is between 600 and 960 pixels. :thinking: